modules/auth/site/config/auth.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I know this is for test purpose, but I would suggest to have different email here.
webmaster@example.com

for instance